mw-webapp/src/component/footer/Footer.module.scss

```diff
@@ -1,36 +1,44 @@
 @import "src/styles/variables";
 
-$userNameFontSize: 16px;
+$logoHeight: 55px;
+$privacyPaddingTop: 10px;
+$privacyPaddingBottom: 12px;
+$iconSize: 32px;
+$privacyLineHeight: 140%;
 
 .footer {
   background-color: var(--quaternaryBgColor);
+  margin-inline: $paddingMedium;
 }
 
 .footerWrapper {
   width: 100%;
   max-width: $maxWidthContainer;
+  align-items: flex-start;
   justify-content: space-between;
-  padding: $paddingBig $paddingMedium;
   border-bottom: $primaryBorder solid var(--secondaryStrokeColor);
   margin: 0 auto;
+  padding-block: $paddingBig;
 }
 
 .privacyLinks {
   width: 100%;
   max-width: $maxWidthContainer;
-  padding: $paddingBig $paddingMedium;
   margin: 0 auto;
   gap: $gapLarge;
+  padding-block: $privacyPaddingTop $privacyPaddingBottom;
 }
 
 .privacyLink {
   color: var(--secondaryTextColor);
   font-size: $fontSizeBig;
+  line-height: $privacyLineHeight;
```

Comment:
All one line texts we use without line-heigth (auto) Discussed about this with designers.

Reply:
Delete please line height for one line texts

```suggestion
```